csharp4python/classes-and-objects-encapsulating-data/README.md

```diff
@@ -89,19 +89,19 @@ josh.Name = "Josh";
 Console.WriteLine(josh.Name);
 ```
 
-When you use properties in this way, the get/set methods are called implicitly when assigning or reading the property.
+When you use properties in this way, the get/set methods are called implicitly when assigning or reading the property, therefore parentheses are not needed.
```

Comment:
Mentioning the lack of parentheses here because, even though the C# documentation refers to accessors as methods, they *look* like properties.